src/definitions.jl

```diff
@@ -438,6 +438,16 @@ end
 
 
 
+"""
+    rheoconv(t)
```

Comment:
How would you feel about `rheoconvert` instead of `rheoconv`. We use `conv` elsewhere for convolutions and I thought initially that the function would be related to that.

Reply:
Agreed. Although it is an exported function (I think - to be checked), I don't think we use it much or promote it, so it should be fine to change the name.